edk2toolext/environment/extdeptypes/nuget_dependency.py

```diff
@@ -61,7 +61,7 @@ def normalize_version(version):
 
         # Remove extra trailing zeros (beyond 3 elements).
         if len(version_parts) == 4 and version_parts[3] == 0:
```

Comment:
I'm not sure. I didn't write this. Let me take a look

Reply:
Ah- so if we have a version (let's say 5.10.5.0 or 5, 10, 5, 0) that has 4 parts and the last part is zero, we drop the last part. The slicing operators are a little strange since they're not inclusive.

So we get (5, 10, 5)